### ISSUE TITLE
Totalconversion

### ISSUE THREAD

**Issue by arkfrancegaming** [FezVrasta/ark-server-tools#501]
Hi,

when i tried to install a total conversion mod (primitiv +) 

ark_TotalConversionMod="504360469" in my arkmanager.cfg

arkmanager useconfig pvp installmod 504360469

but it download it for every server and say that is required but in my others cfg file there are only that command 
#ark_TotalConversionMod="496735411"   

When i add an #ark_TotalConversionMod="504360469" 

the others servers ask for this mod and auto download it, weird ? it seems that the total conversion command is related to all servers

**Comment by klightspeed** [comment]
Settings in `arkmanager.cfg` apply to all instances.

**Comment by arkfrancegaming** [comment]
The mod Id line doesn't do that for all instance, only the total conversation line for me

**Comment by klightspeed** [comment]
Settings in `$instance.cfg` __override__ settings in `arkmanager.cfg`, so if you are specifying a `ark_GameModIds` setting in all of your instance configs, then that setting in `arkmanager.cfg` will not be used.

**Comment by arkfrancegaming** [comment]
Ok but I just want to add a total convertion mod to just one cfg, but all the "use configuration"  cfg are asking for it but I've just add it for one
For exemple, when I use the mod id, all the server cfg got their respective mods, they don't ask for other servers cfg mods

**Comment by arkfrancegaming** [comment]
I haven't that problem before

**Comment by klightspeed** [comment]
If you just want the total conversion mod to be on one instance, then __only__ put it in that instance config.  __Do not__ put it into `arkmanager.cfg`.

**Comment by arkfrancegaming** [comment]
ok so now i've a main.cfg instance with 

arkserverroot="/home/SHI"                                           # path of your ARK server files (default ~/ARK)
serverMap="Theisland"                                               # server map (default TheIsland)
#serverMapModId="515195633"                                         # Uncomment this to specify the Map Mod Id (<fileid> in http://steamcommunity.com/sharedfiles/filedetails/?id=<fileid>)
ark_TotalConversionMod="504360469"                                 # Uncomment this to specify a total-conversion mod
ark_RCONEnabled="True"
ark_RCONPort="27025"  
ark_Port="7777"                                                     # ARK server port (default 7778)
ark_QueryPort="27015"
ark_SessionName="ARK FRANCE OFFICIEL - PvP x7 Primitive +"                                  # if your session name needs special characters please use the .ini instead
ark_ServerPassword=""                                              # ARK server password, empty: no password required to login
ark_ServerAdminPassword="******"                                    # ARK server admin password, KEEP IT SAFE!
ark_MaxPlayers="70"
ark_GameModIds="684959100,558079412"                                         # Uncomment to specify additional mods by Mod Id separated by commas
#ark_AltSaveDirectoryName="SotF"    
#arkflag_OnlyAdminRejoinAsSpectator=true                            # Uncomment to only allow admins to rejoin as spectator
#arkflag_DisableDeathSpectator=true                                 # Uncomment to disable players from becoming spectators when they die
arkflag_log=True
arkflag_BattlEye=""
arkflag_servergamelog=true
ark_ForceAllStructureLocking=true

**Comment by klightspeed** [comment]
Assuming `main` is the instance in which you wish to install the total conversion mod, any one of the following commands:
```
arkmanager useconfig main installmod 504360469
arkmanager installmod @main 504360469
arkmanager useconfig main update --update-mods
arkmanager update @main --update-mods
```

**Comment by arkfrancegaming** [comment]
When is shows that, is it coming from the workshop ?

steam@ns3013274:/root$ arkmanager useconfig main installmod 50460469
Running command 'installmod' for instance 'main'
Downloading mod 50460469 ... Mod 50460469 was not successfully downloaded

**Comment by klightspeed** [comment]
What version are you using - this should be fixed in #486 (currently in 1.6-dev.unstable)
